app-backend/db/src/main/scala/filters/Filterables.scala

```diff
@@ -425,27 +425,30 @@ trait Filterables extends RFMeta with LazyLogging {
     }
 
   implicit val annotationProjectQueryParametersFilter
-    : Filterable[Any, AnnotationProjectQueryParameters] =
+      : Filterable[Any, AnnotationProjectQueryParameters] =
     Filterable[Any, AnnotationProjectQueryParameters] {
       params: AnnotationProjectQueryParameters =>
-        val taskStatusF =
-          params.projectFilterParams.taskStatusesInclude.toList map { status =>
-            val statusString = status.toString
-            statusString.size match {
-              case 0 => None
-              case _ => Some(fr"(ts.summary ->> ${statusString}) > '0'")
+        val taskStatusList =
+          params.projectFilterParams.taskStatusesInclude.toList
+        val taskStatusF = taskStatusList.size match {
+          case 0 => None
+          case _ =>
+            val statusFilterF = taskStatusList map { status =>
+              Some(fr"(ts.summary ->> ${status.toString}) > '0'")
             }
-          }
+            Some(
+              Fragment.const("(") ++ Fragments
+                .orOpt(statusFilterF: _*) ++ Fragment.const(")")
+            )
+        }
+
```

Comment:
I think it's practically fine here because we know the list is small, but it's in general not a good idea to call `.size` to figure out if a list is empty (see the `AvoidSizeEqualsZero` warning [here](https://github.com/sksamuel/scapegoat#inspections), which you've sort of cheated around with the `match`). You can see the cost with a pretty simple example:

```scala
@ val small = (0 to 100).toList

@ val big = (0 to 1000000).toList

@ time { big.size }
res3: (Int, concurrent.duration.FiniteDuration) = (1000001, 19251689 nanoseconds)

@ time { small.size }
res4: (Int, concurrent.duration.FiniteDuration) = (101, 16538 nanoseconds)
```

While `taskStatusList.toNel map { statusList => ...}` will get you the same return type while only checking whether the list has a head (which should be constant time with list length I think): https://github.com/typelevel/cats/blob/master/core/src/main/scala/cats/data/NonEmptyList.scala#L463-L467 (`fromList` is what's called with `.toNel`)